dockers/docker-router-advertiser/start.sh

```diff
@@ -14,5 +14,12 @@ fi
 # Generate /etc/radvd.conf config file
 sonic-cfggen -d -t /usr/share/sonic/templates/radvd.conf.j2 > /etc/radvd.conf
 
+# Enusre at least one interface is specified in radvd.conf
+NUM_IFACES=$(grep -c "^interface" /etc/radvd.conf)
```

Comment:
Instead of grep-ing a text file, you could first check by sonic-cfggen -d -v '...', if there are some interfaces:
then you could generate the conf file, and start job.

Reply:
Done.